draft-ietf-sframe-enc.md

```diff
@@ -607,23 +607,22 @@ framework, as described in {{key-management-framework}}.
 
 If the participants in a call have a pre-existing E2E-secure channel, they can
 use it to distribute SFrame keys.  Each client participating in a call generates
-a fresh encryption key. The client then uses
-the E2E-secure channel to send their encryption key to
-the other participants.
+a fresh `base_key` value that it will use to encrypt media. The client then uses
+the E2E-secure channel to send their encryption key to the other participants.
```

Comment:
This is separate from the change at hand, but when I first read this I wondered why the spec needed to say anything about how the base key was distributed to clients. Would it be less restrictive to simply say that base keys are configured out of band by the key management system?

Reply:
This scheme only makes sense in a situation where there's some mechanism to securely distribute `base_key` values.  So we need some text like this to cue it up.

Reply:
Maybe this could be provided as an example? That is,

```
For example, each client participating... The client could then use the E2E-secure channel..."
```

Reply:
My point is that you can't use this scheme *without* an E2E-secure channel to distribute keys.  So making it an example doesn't make sense.